package.json

```diff
@@ -492,7 +492,7 @@
                 },
                 {
                     "command": "vscode-docker.registries.copyImageDigest",
-                    "when": "view == dockerRegistries && viewItem =~ /registryV2Tag/i",
+                    "when": "view == dockerRegistries && viewItem =~ /commontag/i && !(viewItem =~ /commontag;dockerhub/i)",
```

Comment:
```suggestion
                    "when": "view == dockerRegistries && viewItem =~ /commontag/i && !(viewItem =~ /commontag;.*dockerhub/i)",
```

Reply:
This applies also to line 515, 520, 525, 530, 540

Reply:
looks good. we currently know that there isn't any context value between commontag and dockerhub, so is this for if we add something in the future?

Reply:
Yep, exactly